app/scripts/reducers/api-version.js

```diff
@@ -0,0 +1,35 @@
+import {
+  API_VERSION,
+  API_VERSION_ERROR,
+  API_VERSION_COMPATIBLE,
+  API_VERSION_INCOMPATIBLE
+} from '../actions';
+
+export const initialState = {
+  versionNumber: '',
+  isCompatible: false,
+  warning: ''
+};
+
+export default function reducer (state = initialState, action) {
+  state = {...state};
+  switch (action.type) {
+    case API_VERSION:
+      state.versionNumber = action.payload.versionNumber;
+      state.warning = '';
+      break;
+    case API_VERSION_ERROR:
+      state.apiVersion = action.payload.error.message;
+      state.warning = 'Failed to acquire Cumulus API Version';
```

Comment:
Do we want to warn the user if this check failed or just console.log it? If we do want to show the user, do we want to say something about how the compatibility cannot be determined?

Reply:
Currently this warning is shown on the front page just under the "Cumulus API Version."